src/actions/peers.js

```diff
@@ -31,20 +32,22 @@ export const activePeerSet = data =>
     const config = data.network || {};
 
     if (config.address) {
-      const { hostname, port, protocol } = new URL(addHttp(config.address));
+      // const { hostname, port, protocol } = new URL(addHttp(config.address));
 
-      config.node = hostname;
-      config.ssl = protocol === 'https:';
-      config.port = port || (config.ssl ? 443 : 80);
+      // config.node = hostname;
+      // config.ssl = protocol === 'https:';
+      // config.port = port || (config.ssl ? 443 : 80);
+      config.nodes = [addHttp(config.address)];
     }
     if (config.testnet === undefined && config.port !== undefined) {
       config.testnet = config.port === '7000';
     }
     if (config.custom) {
-      getNethash(Lisk.api(config)).then((response) => {
-        config.testnet = response.nethash === netHashes.testnet;
-        if (!config.testnet && response.nethash !== netHashes.mainnet) {
-          config.nethash = response.nethash;
+      const getNethash = new Lisk.APIClient(config.nodes, config.nethash, {});
+      getNethash.node.getConstants().then((response) => {
```

Comment:
`getNethash` is not describing what is stored in the variable, it is more something like `apiClientInstance`